pages/attributes.vue

```diff
@@ -0,0 +1,342 @@
+      :class="paragraph"
+    >Notice that this method can be called before and after 
+    the creation of Attributes associated to the 
+    <span v-html="attributeManager" /> 
+    (as done at the fourth line of the above code). 
+    </p>
+
+    <p
+      :class="paragraph"
+    >As in standard library, there is also a method to reserve Attributes capacities. 
+    </p>
+
+    <p
+      :class="paragraph"
+    >Now, take a look at the third line of the above code. 
+    This line creates an attribute using the method <code>find_or_create</code>.
+    Four elements are needed to define the Attribute to create:
+    </p>
+
+    <ul :class="paragraph">
+      <li>
+        An <strong>attribute name</strong> (here <code>first_attribute</code>): 
+        the first parameter of the method <code>find_or_create</code>;
+      </li>
+      <li>
+        The <strong>type</strong> of Attribute storage (here a 
+        <span v-html="variableAttribute" />): 
+        the first template parameter of the method (more details below);
+      </li>
+      <li>
+        The <strong>type</strong> of Attribute values (here <code>double</code>): 
+        the second template parameter of the method;
+      </li>
+      <li>
+        The <strong>default value</strong> (here <code>12.3</code>): 
+        the second parameter of the method.
+      </li>
+    </ul>
+
+    <p
+      :class="paragraph"
+    >If the attribute already exists and has the same name, 
+    same storage and same value type, 
+   a pointer to the pre-existing attribute is returned
+    without creating a new attribute. If the attribute name already exist but 
+    with a different storage or a different value type, an exception is thrown.
+    </p>
+
+    <h3 :class="subsection">Attribute Storage</h3>
+
+    <p
+      :class="paragraph"
+    >Each attribute may be stored using a storage adapted to its use and its values.
+    In OpenGeode, there are three types of attribute storage, 
+    but you can easily define your own.
+    These storages are:</p>
+
+    <ul :class="paragraph">
+      <li>
+        <span v-html="variableAttribute" />: 
+        one value is stored per attribute element.
+        The number of stored values is equal to the attribute size;
+      </li>
+      <li>
+        <span v-html="constantAttribute" />:  
+        all the attribute elements have the given default value as value.
+        Only one value is stored (the default value);
+      </li>
+      <li>
+        <span v-html="sparseAttribute" />:  
+        all the attribute elements have the given default value as value,
+        excepted the ones that have been explicitly defined.
+        The number of stored values is between one (only the default value) 
+        and the attribute size plus one 
+        (one value for each attribute element and the default value).
+        This storage is useful if the number of values different from the 
+        default value is small with regards to the attribute size.
```

Comment:
Add a better idea on what "small" means: is it less than 50%, less than 90%??

Reply:
This kind of user is up to the user context. And I think there is some benchmarking to do to answer this question